docs/src/guides/run-analysis-on-terra.rst

```diff
@@ -2,43 +2,47 @@
 
-1. `Setup a Terra account <https://terra.bio/>`_
-#. Navigate to Dockstore: `ncov:master`_
-#. Top right corner, under **Launch with**, click on **Terra**
-#. Under "Workflow Name" set a name, can also leave default ``ncov``, and select your **Destination Workspace** in the drop down menu.
-#. Click button **IMPORT**
-#. In your workspace, click on the **WORKFLOWS** tab and verify that the imported workflow is showing a card
-
-.. _`ncov:master`: https://dockstore.org/workflows/github.com/nextstrain/ncov:master?tab=info
+1. `Set up a Terra account <https://terra.bio/>`_.
+2. Navigate to Dockstore: `nextstrain/ncov/ncov <https://dockstore.org/workflows/github.com/nextstrain/ncov/ncov:master?tab=info>`_
+3. At the top right corner, under **Launch with**, click on **Terra**. You may be prompted to log in.
+4. Provide a **Workflow Name** (e.g. ``ncov``).
+5. Select a **Destination Workspace** from the dropdown menu.
+6. Click **IMPORT**.
+7. In your workspace, click on the **WORKFLOWS** tab and verify that the imported workflow is showing a card.
 
 Upload your data files into Terra
 =================================
 
-1. Navigate to: `https://app.terra.bio/#upload`_.
+1. Navigate to: `https://app.terra.bio/#upload <https://app.terra.bio/#upload>`_.
 
-#. Select your workspace
-#. At the top, hit the **+** button to "create a collection"
-#. Within the collection, at bottom right, click **+** button to upload file, or drag and drop files to upload them.
-#. Go back to your Terra Dashboard
-#. Click on the **DATA** tab
-#. On the left, under **OTHER DATA**, click **Files** and there should be an "uploads/" folder shown to the right
-#. Click on "uploads/" to view your collection and verify that your files have been uploaded
-
-.. _`https://app.terra.bio/#upload`: https://app.terra.bio/#upload
+2. Select your workspace
+3. At the top, hit the **+** button to "create a collection"
+4. Within the collection, at bottom right, click **+** button to upload file, or drag and drop files to upload them.
+5. Go back to your Terra Dashboard
+6. Click on the **DATA** tab
+7. On the left, under **OTHER DATA**, click **Files** and there should be an "uploads/" folder shown to the right
+8. Click on "uploads/" to view your collection and verify that your files have been uploaded
 
 Connect your data files to the WDL workflow
 ===========================================
 
 1. On the **DATA** tab, click on **+** next to the **TABLES** section to create a Data Table
-#. Download the "sample_template.tsv" file
-#. Create a tab delimited file similar to below:
+2. Download the "sample_template.tsv" file
+3. Create a tab delimited file similar to below :download:`example script <./ncov_examples.tsv>`:
 
 ::
 
     entity:ncov_examples_id	metadata	sequences	configfile_yaml
+    blank   
     example	gs://COPY_PATH_HERE/example_metadata.tsv	gs://COPY_PATH_HERE/example_datasets/example_sequences.fasta.gz
     example_build		gs://COPY_PATH_HERE/example-build.yaml
```

Comment:
Suggestion: Add instructions on how to populate these paths. My assumptions:

1. `configfile_yaml`: This should reference a [workflow config file](https://docs.nextstrain.org/projects/ncov/en/latest/reference/glossary.html#term-config-file) uploaded to workspace data files. The `gs://` path can be copied from links on the Workspace Data Files view.
2. `metadata`: This should reference a metadata TSV file. (How do you reference the ingest workflow result, is it something like `workspace.genbank_metadata_tsv`?)
3. `sequences`: This should reference a sequences FASTA file.